docs/Community/governance/README.md

```diff
@@ -0,0 +1,40 @@
+Open Enclave Community Governance Committee
+===========================================
+
+The [charter](charter.md) defines the scope and processes by which the CGC operates.
+
+Meetings: public CGC meetings are held on Wednesdays (bi-weekly) (*link TBD*)
+
+Minutes: aka.ms/openenclave/cgc
+
+How to raise a topic to the CGC
+-------------
+
+TODO
```

Comment:
Ties into a comment of mine above where we may want to have an email which anyone can send messages to the CGC (so like oesdkcgc but public).